src/environments/environment.regression.ts

```diff
@@ -18,7 +18,7 @@ export const environment = {
       initUri: 'https://fundraise.charitycheckouttest.co.uk/api/checkout/init',
     },
     stripe: {
-      enabled: false,
+      enabled: true,
```

Comment:
@NoelLH Isn't this going to clash with Enthuse if both are `true` ?

Reply:
@tbg-jasonfung no, see https://thebiggive.atlassian.net/browse/DON-254 – We need both to be true. Have a look at `DonationStartComponent.handleCampaign()` for how this is used.